hydromt_fiat/fiat.py

```diff
@@ -274,13 +274,16 @@ def setup_vulnerability(
         if step_size:
             self.set_config("vulnerability.step_size", step_size)
 
-    def setup_vulnerability_from_csv(self, csv_fn: Union[str, Path], unit: str) -> None:
+    def setup_vulnerability_from_csv(self, csv_fn: Union[str, Path], vulnerability_identifiers_and_linking_fn: str, unit: str) -> None:
```

Comment:
This is not something you changed, but we typically use `_fn` to refer to a filename or filepath. Here `csv_fn` links to a directory path, `csv_dir` would be more intuitive.